### DISCUSSION TITLE
Vanity URL to emulate Dokuwiki functionality?

### DISCUSSION THREAD

**Discussion by issmirnov** [outline/outline#2707] [Self Hosting] [answered]
Hi folks,

I'm doing my annual exploration where I look for https://www.dokuwiki.org/dokuwiki alternatives. I discovered Outline, and the overall polish is really awesome.

I use my wiki both for private notes and for publicly shared sections. Is it possible to share pages with vanity urls? Eg, `https://my.wiki/research/cars`.

I see that I can publish pages, but they generate a UUID. Curious if this is possible.

Thanks in advance!

**Comment by tommoor** [comment]
Hi Ivan, this is not currently possible

**Comment by tommoor** [comment, marked as answer]
This is possible today, click on the settings cog when sharing to choose the slug.
